services/clojars/clojars-downloads.tester.js

```diff
@@ -0,0 +1,29 @@
+'use strict'
+
+const Joi = require('joi')
+const createServiceTester = require('../create-service-tester')
+
+const t = createServiceTester()
+module.exports = t
+
+t.create('clojars downloads (valid)')
+  .get('/prismic.json')
+  .expectJSONTypes(
+    Joi.object().keys({
+      name: 'downloads',
+      value: /^[0-9]+$/,
+    })
+  )
+
+t.create('clojars downloads (lots)')
+  .get('/prismic.json')
+  .expectJSONTypes(
```

Comment:
I'm not clear on the difference between these two tests - it seems like there should only be one. I think you can just call `/prismic.json` and use the `isMetric` test validator to check the output:

https://github.com/badges/shields/blob/510491f37644432740548afd8cdbac7e1978b59f/services/test-validators.js#L60

(require it from `test-validators` )